Templates/Per Tenant Extension/.github/workflows/CreateRelease.yaml

```diff
@@ -1,4 +1,5 @@
 name: ' Create release'
+run-name: "Create release - Version ${{ inputs.tag }}"
```

Comment:
doesn't the run-name section has to be after the workflow-dispatch section?

Reply:
This worked just fine for me 🤷‍♂️ 

Here's an example (that failed but for a different reason I think 😆)
https://github.com/aholstrup1/BCApps/actions/runs/8371584108